Dockerfile

```diff
@@ -313,6 +313,20 @@ ENV GO111MODULE on
 ARG TESTPKGS
 RUN --mount=type=cache,target=/root/.cache/go-build go test -v -count 1 -race ${TESTPKGS}
 
+# The integration.test target builds integration test binary.
+
+FROM base AS integration.test-build
```

Comment:
nit: can we use `-` instead of `.` to align with the convention?

Reply:
yep, sure, I followed the default naming for `go test -c`, but we can certainly make it all dashes